tests/performance-test/performance-test.sh

```diff
@@ -1,23 +1,21 @@
 #!/bin/bash
 set -e
 
+
 usage(){
     cat << ENDUSAGE
 Runs on the dev/CI machine to execute a performance test and abstracts between
 running collectd-tg (tg) or telemetry-bench (tb).
-
```

Comment:
I can see we have stylistic differences. You win this time!

Reply:
I actually prefer the spacing you had in there, it must have been deleted when I resolved conflicts. Adding it back in..